src/invokeArgs.js

```diff
@@ -0,0 +1,57 @@
+import { curryN } from 'ramda';
+
+import isNotArray from './isNotArray';
+import isNotObj from './isNotObj';
+import isNotFunction from './isNotFunction';
+
+/**
+ * Invokes the method at path of object with given arguments.
+ *
+ * @func invokeArgs
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.27.0|v2.27.0}
+ * @category Object
+ * @sig Array -> Array -> Object -> *
+ * @param {Array.<string|number>} path The path of the method to invoke
+ * @param {Array} args The arguments to invoke the method with
+ * @param {Object} obj The object to query
+ * @return {*}
+ * @example
+ *
+ * RA.invokeArgs(['abs'], [-1], Math); //=> 1
+ * RA.invokeArgs(['path', 'to', 'non-existent', 'method'], [-1], Math); //=> undefined
+ */
+
+const checkArgs = (pathToMethod, args) => {
+  return isNotArray(pathToMethod) || isNotArray(args);
+};
+
+const invokeArgs = curryN(3, (pathToMethod, args, obj) => {
+  if (checkArgs(pathToMethod, args)) return undefined;
+
+  /**
+   * If we need these test cases:
+   *  assert.strictEqual(RA.invokeArgs([], [1, 2], testFunc), 3);
+   *  assert.strictEqual(RA.invokeArgs(['methodWithoutArgs'], [], testObj), 1);
+   * then arrays 'pathToMethod' and 'args' can be empty, so we don't need isEmptyArray
```

Comment:
If `path` is an empty array then we should do nothing. We have to take care of this corner case because  when we use `R.path` for the implementation, if provided with empty array as a first arugment, return original value provided as second argument.

```js
const obj = {};
const method = R.path([], obj);
assert.true(method === obj)

Reply:
OK, I get it.